shrike/build/__init__.py

```
@@ -7,4 +7,7 @@
 Azure Machine Learning workspaces.
 """
 
-from .commands import prepare, register
+try:
+    from .commands import prepare, register
+except ModuleNotFoundError as error:
+    print(f"{error.msg}. Please install using `pip install shrike[build]`.")
```

Don't just "swallow" the exception. Re-throw.

Same below.